horace_core/sqw/file_io/@sqw_file_interface/sqw_file_interface.m

```diff
@@ -89,6 +93,14 @@
         function npix = get.npixels(obj)
             npix = obj.npixels_;
         end
+        function pix_size = get.pixel_size(~)
+            % 4 bytes x 9 columns -- default pixel size.
+            %
+            % TODO:
+            % No other size is currently supported by loaders. If this, one
+            % day, changes, the class hierarchy should be tinkered with.
+            pix_size = 4*9;
```

Comment:
Again these should probably be constants in the `PixelData` class rather than hard-coded numbers.

Reply:
In addition to above, this is file reader intended to read 4x9 pixels. This thing is hardcoded in more then one place, including `fread(...,'float32*')` operation. PixelData is another class, which contains pixels (BTW, 8x9 pixels). The fact they have similar constants is entirely accidental. Not accidental is the fact that PixelData knows how to get its 8x9 pixels using sqw_file_interface. Only confusion will be added if `PixelData` constants are used by `fileIO` classes